src/providers/fail_over.c

```diff
@@ -1188,9 +1189,18 @@ fo_resolve_service_done(struct tevent_req *subreq)
                                     &common->rhostent);
     talloc_zfree(subreq);
     if (ret != EOK) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "Failed to resolve server '%s': %s\n",
+        if (resolv_status == ARES_EFILE)
+        {
+            /* resolv_strerror(resolv_status) provided msg from c-ares lib.
+             * c-ares lib in most distros will default to /etc/hosts file. */
```

Comment:
I think more precise description would be something like:
`c-ares lib in most distros will default to /etc/hosts for file based host resolving`

Reply:
H Pawel, 

Thank you for your feedback. I have made the necessary changes. Can you review again. Thanks